ecosystem/sep-00xx.md

```diff
@@ -0,0 +1,131 @@
+## Preamble
+
+```
+SEP: 00xx
+Title: Stellar Web Authorization
+Author: Leigh McCulloch <@leighmccullcoh>
+Status: Draft
+Created: 2019-11-22
+Version 1.0.0
+```
+
+## Simple Summary
+
+This SEP defines the standard way for clients such as wallets or exchanges to expand authenticated web sessions created with SEP-10 with additional claims about control over Stellar accounts to facilitate authorization of a specific Stellar account. SEP-10 provides a JWT that proves possession of a Stellar key. This SEP provides additional claims that can be added to the JWT that prove a weight of control of a Stellar account, and a process that a client and server can follow to augment a SEP-10 JWT with those claims.
+
+## Abstract
+
+This protocol is the definition of additional claims an implementer of SEP-10 can include in the JWT that capture at a point in time the weight of control a Stellar key, or multiple keys, has for a given account.
+
+The authorization flow is as follows:
+
+1. The client obtains a [JWT](jwt.io) using SEP-10 proving possession of one or more Stellar keys.
+1. The client calls the server with the SEP-10 JWT requesting authorization of a Stellar account.
+1. The server verifies that the keys in the SEP-10 JWT are signers of the Stellar account.
+1. The server responds with the JWT augmented with additional claims representing the account and weighted control of the keys.
+1. Any future calls to the server can be authenticated by including the JWT as a parameter and a level of control of the account can be authorized by the additional claims.
+
+The flow achieves several things:
+
+* The server can verify that the client holds the secret key(s) that can sign for an account without checking the network on every request containing a SEP-10 JWT
+* The server can chose a timeout for the user's proof that they are a signer of a Stellar account independent of the timeout that they possess their secret keys
+* Since the signers of a Stellar account can change at anytime the server can choose a small timeout of JWTs of this SEP, while keeping a longer timeout of JWTs of SEP-10. A longer lasting SEP-10 JWT can be used to check and refresh proof of control of an account without the user needing to resign transactions.
+
+## Authentication Endpoint
+
+A web service indicates that it supports account authorization via this protocol by specifying `WEB_AUTHZ_ENDPOINT` in their [`stellar.toml`](sep-0001.md) file. This is how a wallet knows where to find the authorization server. A web server is required to implement the following behavior for the web authorization endpoint:
+
+* [`POST <WEB_AUTH_ENDPOINT>`](#authorize): exchange a SEP-10 key JWT for an account JWT
```

Comment:
s/WEB_AUTH_ENDPOINT/WEB_AUTHZ_ENDPOINT

Reply:
also why AUTHZ?

Reply:
This new endpoint will be doing an authorization check. Authorization is sometimes abbreviated to `authz`. Authentication is sometimes abbreviated to `authn` when not wanting the abbreviation to be confused with authorization.